lib/cs_guide_web/templates/venue/show.html.eex

```diff
@@ -8,15 +8,9 @@
         style="background-image: url('https://s3-eu-west-1.amazonaws.com/<%=Application.get_env(:ex_aws, :bucket)%>/<%= img.entry_id %>');">
       </div>
     <% else %>
-      <%= if @venue.external_image do %>
-        <div class="db bg-venue w-100"
-          style="background-image: url('<%= @venue.external_image %>');">
-        </div>
-      <% else %>
-        <div class="db bg-venue w-100"
-          style="background-image: url('http://www.camera-m.com/img/placeholder-image-wide.png');">
-        </div>
-      <% end %>
+      <div class="db bg-venue w-100"
+        style="background-image: url('http://www.camera-m.com/img/placeholder-image-wide.png');">
```

Comment:
I would prefer if this image was "re-uploaded" e.g. by copy-pasting it into GitHub so that we are not relying on someone else to host (_and potentially "break"_) the images for the CS app/site.

Reply:
@nelsonic this is not our image and so I'm not sure if we have the right to use it. I think it may be better for us to ask Harry for a new image altogether...

Reply:
@Cleop I _understand_ your concern regarding copyright it's _good_ to be cautious. 
But linking to an image on someone else's server is also "bandwidth theft". 
see: https://en.wikipedia.org/wiki/Inline_linking

The image we are using: http://www.camera-m.com/img/placeholder-image-wide.png 
is insecure. (_served over HTTP not HTTPS_) so we should re-upload it in _any_ case. 

![image](https://user-images.githubusercontent.com/194400/49531010-48dad180-f8b1-11e8-8d89-1e61320e1d82.png)

Also, the image is not very sophisticated ... we could probably re-create it in a graphics app in 5 mins.
(_and not have to bother Harry ..._)

Please see: https://github.com/dwyl/learn-tachyons#how-to-create-an-image-placeholder
Thanks. ✨